internal/apis/config/controller/v1alpha1/defaults.go

```diff
@@ -243,9 +241,7 @@ func SetDefaults_ControllerConfiguration(obj *v1alpha1.ControllerConfiguration)
 		obj.PprofAddress = defaultProfilerAddr
 	}
 
-	if obj.Logging == nil {
-		obj.Logging = defaultLogging
-	}
+	logsapi.SetRecommendedLoggingConfiguration(&obj.Logging)
```

Comment:
This change fixes an issue that caused the `FlushFrequency` to be set to 0 in case you specify any other logging option.